internal/output/templates/html.tmpl

```diff
@@ -256,6 +256,9 @@ iVBORw0KGgoAAAANSUhEUgAAAMAAAADACAMAAABlApw1AAAABGdBTUEAALGPC/xhBQAAAAFzUkdCAK7O
   </head>
 
   <body>
+    {{- if .RunQuotaExceeded }}
+      <p>Infracost free monthly run limit exceeded. Please contact <a href="mailto:hello@infracost.io">hello@infracost.io</a> to purchase usage pack. </p>
```

Comment:
I think we probably don't want to hardcode this message in the CLI if we can avoid it?

Reply:
so where will the messge come from? The `error` field returned by the pricing api?